packages/big-design/src/managers/alerts/AlertsManager.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useState } from 'react';
+
+import { Alert, AlertProps } from '../../components';
+
+import { alertsManager } from './manager';
+
+export const AlertsManager: React.FC = ({ children }) => {
+  const [alert, setAlert] = useState<AlertProps | null>(null);
+
+  useEffect(() => {
+    alertsManager.subscribe(setAlert);
+
+    return () => {
+      alertsManager.unsubscribe();
+    };
+  }, []);
+
+  return (
+    <>
+      {children}
```

Comment:
I don't think we should make this a wrapper, can we make it so an app just includes `<AlertsManager />` without having to wrap the whole App?